app/controllers/spaces_controller.rb

```diff
@@ -42,7 +42,7 @@ def create
   def update
     if space.update(space_params)
```

Comment:
`space.update` is failing and doing a roll back. But I do not see any error indicating why. 

![Screenshot 2023-06-28 at 8 27 52 PM](https://github.com/zinc-collective/convene/assets/16140873/4cee9277-1949-415c-9e83-c7911123bb13)

Reply:
You wouldn't see the errors in the server log this way. What you want to do is inspect `space.errors` after the failed update in some way. The typical Rails way is that you have an `else` branch that you use to display the errors when the update is not successful.

Reply:
Ideally, the form would also show any validation errors underneath of the field (you can see how to do this by looking at  [`app/views/application/_text_field.html.erb`](https://github.com/zinc-collective/convene/blob/main/app/views/application/_text_field.html.erb), which shows how to use the `_error` partial for this.